debian/postinst

```diff
@@ -2,15 +2,23 @@
 
 set -e
 
+ESM_KEY_ID="56F7650A24C9E9ECF87C4D8D4067E40313CB4B13"
```

Comment:
Does this need to be different per-series?

Reply:
Yes it does, I had flied a separate issue for this to avoid bloating this branch more, but if you agree I'll pull that work into this branch as this branch doesn't work without a followup for #692

Reply:
I think it probably is worth doing here; how much bloat would you expect it to add? (Unless I'm missing something, this is just a case statement, right?)

Reply:
Just missing that we'll have to parameterize a few of the functions in postinst to handle upgrade path trusty->xenial etc. But It's not too much. I'll tease it out today.

Reply:
OK, I was assuming we were only going to make the changes to install these per-series (as this change would otherwise regress that specific functionality); I think the additional changes fixing other, pre-existing issues would be best submitted as a separate PR.

Reply:
Ok just backed out xenial/bionic support for esm from postinst. Filed a separate PR to fix #692. That PR is #699